primitives/src/v1.rs

```diff
@@ -666,6 +680,10 @@ sp_api::decl_runtime_apis! {
 		fn persisted_validation_data(para_id: Id, assumption: OccupiedCoreAssumption)
 			-> Option<PersistedValidationData<N>>;
 
+		// TODO: Adding a Runtime API should be backwards compatible... right?
```

Comment:
Yeah, it is fine. We don't need to worry about backcompat until we've deployed on Rococo. This TODO needs resolving before merge